stages/test/test_kickstart.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python3
+
+import os.path
+import tempfile
+from dataclasses import dataclass
+
+import pytest
+
+from osbuild.testutil.imports import import_module_from_path
+
+
+@dataclass
+class KsTestCase:
+    """KsTestCase is a single test case for the kickstart stage
+
+    options:  a dict with the options passed to the kickstart stage
+    expected: the expected string in the generated kickstart file
+    """
+    options: str
+    expected: str
+
+
+@pytest.fixture(name="ks_test_cases")
```

Comment:
Let's do this as a `@pytest.mark.parametrize`: https://docs.pytest.org/en/7.3.x/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions instead. We can name the list if we plan to use it for multiple tests :)

Reply:
Thank you, this is excellent! I updated the test and it's indeed much nicer now